### PR TITLE
Added needed versions

### PR DESCRIPTION
**Pull request by agaida**
fixes lxde/libfm-qt/issues/89

### CODE REVIEW

**Comment by agaida** [comment]
@luis-pereira @palinek @tsujan - what do you think? Good to go?

**Comment by tsujan** [comment]
Yes, if they're the required versions -- I haven't had Qt 5.6 for ages ;)

**Comment by agaida** [comment]
lxqt-build-tools need at least 5.6.1 - so it should be sufficient. glib should also be sufficient

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,6 +9,13 @@ set(LXQT_BUILD_TOOLS_MINOR_VERSION 3)
 set(LXQT_BUILD_TOOLS_PATCH_VERSION 2)
 set(LXQT_BUILD_TOOLS_VERSION ${LXQT_BUILD_TOOLS_MAJOR_VERSION}.${LXQT_BUILD_TOOLS_MINOR_VERSION}.${LXQT_BUILD_TOOLS_PATCH_VERSION})
 
+
+# Check for needed versions
+# We need at least Qt 5.6 and glib-2.0 >= 2.50 to build all LXQt parts 
+find_package(PkgConfig REQUIRED)
+find_package(Qt5Core "5.6.1"  REQUIRED)
+pkg_check_modules(GLIB2 glib-2.0>=2.50 REQUIRED)
```
Comment (palinek) [inline CMakeLists.txt]: There is a "find" module for glib done by @luis-pereira already -> https://github.com/lxde/lxqt-build-tools/blob/master/cmake/find-modules/FindGLIB.cmake

Comment (agaida) [inline CMakeLists.txt]: the find module is a little bit overkill - and it would also need two or three lines, so i tried to keep it simple.

Comment (palinek) [inline CMakeLists.txt]: > the find module is a little bit overkill

I thought I was alone who didn't see the need for find modules on top of pkg_config configurations...

Comment (agaida) [inline CMakeLists.txt]: na - it isn't about the need - but in this case it is just: Hey - in need some Qt and som glib in a minimum version. And including the module and call it the right way are two lines too :)

**Comment by agaida** [comment]
and i want it really simple in a notation that everyone know - people do some strange things sometimes like comment out checks for a reason - and i think we should not make their life harder than needed. So we think that it is cool to prevent a build at all when we know that several components will not build. Some other people might think otherwise. So these three lines are a statement: We want it that way and if you ignore our whishes and comment the check out  - please don't file any bugs about, you are alone with your problem.

**Pull request merged**